infra/base-images/base-builder/compile_fuzztests.sh

```diff
@@ -68,16 +78,20 @@ then
   declare -r RSYNC_FILTER_ARGS=("--include" "*.h" "--include" "*.cc" "--include" \
     "*.hpp" "--include" "*.cpp" "--include" "*.c" "--include" "*/" "--include" "*.inc" \
     "--exclude" "*")
+
   project_folders="$(find . -name 'bazel-*' -type l -printf '%P\n' | \
                     grep -v -x -F \
                     -e 'bazel-bin' \
-                    -e 'bazel-out' \
                     -e 'bazel-testlogs')"
   for link in $project_folders; do
    if [[ -d "${PWD}"/$link/external  ]]
    then
      rsync -avLk "${RSYNC_FILTER_ARGS[@]}" "${PWD}"/$link/external "${REMAP_PATH}"
    fi
+   if [[ -d "${PWD}"/$link/k8-opt  ]]
```

Comment:
Likewise here -- can you add more context in comments for future readers of this code?